Advanced-Recruit-Training/src/BatteryStateOfChargeService.h

```diff
@@ -1,28 +1,31 @@
 #pragma once
 
 #include "I_BatteryStateOfChargeService.h"
+#include <QString>
 
 class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
 {
 public:
     explicit BatteryStateOfChargeService(double initialStateOfChargePercent);
     virtual ~BatteryStateOfChargeService();
 
-    /*Returns the total amp hours used*/
+
```

Comment:
Here, and on lines 15, 18 and 21, there are now extra new lines from where the comment were deleted. Let's get rid of them.